cmd/internal/cli/build.go

```diff
@@ -285,6 +287,22 @@ var buildRemoteFlag = cmdline.Flag{
 	Hidden:       true,
 }
 
+var buildVarArgsFlag = cmdline.Flag{
+	ID:           "buildVarArgsFlag",
+	Value:        &buildArgs.buildVarArgs,
+	DefaultValue: []string{},
+	Name:         "build-arg",
+	Usage:        "--build-arg is used for passing variables to build procedure",
```

Comment:
Don't include the variable name in the Usage.  Use somethng like "defines variable=value to replace '{{variable}}' with value in build definition files".

Reply:
thanks! resolved in the latest commit.
https://github.com/apptainer/apptainer/pull/1344/commits/1842bc374a7db1c7a72fb8fa11d811d7de4e2ff4